opm/parser/eclipse/EclipseState/Tables/PvtoTable.hpp

```diff
@@ -17,22 +17,34 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
 #ifndef OPM_PARSER_PVTO_TABLE_HPP
-#define	OPM_PARSER_PVTO_TABLE_HPP
+#define OPM_PARSER_PVTO_TABLE_HPP
 
 #include <opm/parser/eclipse/EclipseState/Tables/PvtxTable.hpp>
 
+#include <array>
+#include <cstddef>
+#include <vector>
+
 namespace Opm {
 
     class DeckKeyword;
 
     class PvtoTable : public PvtxTable {
     public:
+        struct FlippedFVF {
+            std::size_t i;
+            std::array<double, std::size_t{2}> Rs;
+            std::array<double, std::size_t{2}> Bo;
```

Comment:
How about the slightly leaner: `std::array<double, 2> Rs;` - or even named members for `Rs[0]` and `Rs[1]`?